lib/ae_mdw_web/controllers/oracle_controller.ex

```diff
@@ -15,6 +15,15 @@ defmodule AeMdwWeb.OracleController do
 
   @spec oracle(Conn.t(), map()) :: Conn.t()
   def oracle(%Conn{assigns: %{state: state, opts: opts}} = conn, %{"id" => id}) do
+    with opts <- [{:v3, true} | opts],
```

Comment:
Why use `opts` as part of the `with` statement instead of a regular variable binding?

Reply:
A habbit, on an older project we used to put the inside the with and if it's going to always match we used `=` instead of  `<-`. I don't have a strong preference i can change it back